sdk/metric/metrictest/exporter_test.go

```diff
@@ -40,10 +40,10 @@ func TestSyncInstruments(t *testing.T) {
 		fcnt.Add(ctx, 2)
 
 		err = exp.Collect(context.Background())
-		assert.NoError(t, err)
+		requrie.NoError(t, err)
```

Comment:
```suggestion
		require.NoError(t, err)
```

Reply:
Gah, that's my bad on the spelling.

@MadVikingGod I didn't make a comment for each test run, but the intention was to update to require for all the parts that are setting up the test condition.

Reply:
Let me know if I should add a complete suggestion for all the test runs.